includes/admin/class-sensei-admin-notices.php

```diff
@@ -152,6 +182,22 @@ protected function get_notices() {
 	 * @access private
 	 */
 	public function add_admin_notices() {
+		$screen_id = $this->get_screen_id();
+
+		/**
+		 * Adds the ability to hide notices on a specific screen.
+		 *
+		 * @hook sensei_hide_admin_notices_on_screen
```

Comment:
Wrong hook name in comment

Reply:
Good catch! 2bc3d69e54d4d6d05db3d4b26303c1b9a089f6ba